public/js/components/Tabs/_commissionedLengthTestData.js

```diff
@@ -29,7 +29,7 @@ const articles = [
   {headline: "Neurotechnology, Elon Musk and the goal of human enhancement ", path:"technology/2018/jan/01/elon-musk-neurotechnology-human-enhancement-brain-computer-interfaces", wordCount: 1000, commissionedWordCount: 0},
 ]
 
-export default {
+export {
```

Comment:
Why don't you need `default`

Reply:
@susiecoleman `export { ... }` exports a list of named exports. `export default { ... }` exports one object with all of the exports as properties on that object. A subtle difference but, for one thing, you can't destructure an object during an import, just the named exports. So you can only do `import { /* some named import */ } from "..."` wiht_out_ the default!